ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManager.java

```diff
@@ -88,6 +88,11 @@
   public abstract void onBackPressed();
 
   /**
+   * This method will give JS the opportunity to receive intents via Linking.
+   */
+  public abstract void onNewIntent(Intent intent);
```

Comment:
It looks like `onActivityResults` is also going through ReactInstanceManager, then I think this can follow the same route. Never mind, carry on.